src/main/java/org/sunbird/user/registration/service/UserRegistrationServiceImpl.java

```diff
@@ -332,7 +322,6 @@ private UserRegistration getRegistrationObject(UserRegistrationInfo userRegInfo)
 		userRegistration.setMapId(userRegInfo.getMapId());
 		userRegistration.setOrganisationType(userRegInfo.getOrganisationType());
 		userRegistration.setOrganisationSubType(userRegInfo.getOrganisationSubType());
-		userRegistration.setPhone(userRegInfo.getPhone());
```

Comment:
why this statement is removed - this is going to affect direct user registration flow.